setup.py

```diff
@@ -3,21 +3,21 @@
 # from https://stackoverflow.com/a/9079062
 import sys  
 if sys.version_info[0] < 3:
-    raise Exception("The fenicsadapter only supports Python3. Did you run $python setup.py <option>.? Try running $python3 setup.py <option>.")
+    raise Exception("The fenicsprecice only supports Python3. Did you run $python setup.py <option>.? Try running $python3 setup.py <option>.")
 
 # using point 3. in https://packaging.python.org/guides/single-sourcing-package-version/#single-sourcing-the-version
 version = {}
-with open("fenicsadapter/version.py") as fp:
+with open("fenicsprecice/version.py") as fp:
     exec(fp.read(), version)
 
-setup(name='fenicsadapter',
+setup(name='fenicsprecice',
       version=version['__version__'],
-      description='preCICE-adapter for the open source computing platform FEniCS ',
+      description='preCICE-adapter for the open source computing platform FEniCS',
```

Comment:
```suggestion
      description='FEniCS-preCICE adapter: A preCICE-adapter for the open source computing platform FEniCS.',
```

Reply:
Following the suggestion in https://github.com/precice/fenics-adapter/pull/93#discussion_r510771521, I would use this `description`.